### DISCUSSION TITLE
Importing sionna

### DISCUSSION THREAD

**Discussion by pablosreyero** [NVlabs/sionna#212] [Q&A]
Hello everyone!

I hope you are all doing good. I have been trying for quite a long time, to install sionna in a virtual machine with Ubuntu 20.04. I've ran into all kind of errors, from dying kernels while trying to import sionna or tensorflow, to this last error, which states the following:

`ImportError: cannot import name 'deserialize_keras_object' from partially initialized module 'keras.saving.legacy.serialization' (most likely due to a circular import) (/home/pablo/anaconda3/envs/testenv/lib/python3.9/site-packages/keras/saving/legacy/serialization.py)`

My working environment is the following:
-Virtual Machine with Ubuntu 20.04, 5 cores and 16 Gb of RAM.
-Anaconda environment with Python 3.9.17
-Tensorflow 2.12.0
-Sionna 0.15.1
-Numpy 1.23.5

Any help will definetely be appreciated, I've tried everything and I'm not able to get rid of this error. It is also very weird, because I'm complying with all basic requirements that are stated in the installation process at the web page.

Muchas gracias and kind regards,

Pablo.-

**Comment by merlinND** [comment]
Hello @pablosreyero,

Could you please share the full Python code which triggers this error?

**Reply by pablosreyero** [reply]
Hi Merlin, thanks for your reply, 

I don't even have any developed code yet, it is just the import section that automatically triggers this error. On the following screenshots, you can see the error in a jupyternotebook and in the Ubuntu terminal.

![image](https://github.com/NVlabs/sionna/assets/104429807/07826290-7da1-481a-88d3-9e706a1c79b4)

![image](https://github.com/NVlabs/sionna/assets/104429807/0857b78a-afd9-408b-9065-85418591c561)

Thanks for your help.
